group_vars/all.yml

```diff
@@ -119,7 +120,7 @@ inspection_iprange: 192.168.24.110,192.168.24.250
 # undercloud to access overcloud resources
 external_gateway: 172.17.5.1/24
 external_network_vlan_id: 300
-clean_nodes: False
+clean_nodes: false
```

Comment:
why is this false?   Is it to speed up tests that don't involve ceph?   Without clean_nodes: true, you won't be able to do more than 1 deployment with Ceph, because Ceph will detect that data is already living on the storage devices and won't reinitialize it.

Reply:
Yes, by default clean_nodes set to false so that user can enable cleaning based on the need as per the recommendations